npm/react/.releaserc.js

```diff
@@ -1,6 +1,3 @@
 module.exports = {
   ...require('../../.releaserc.base'),
-  branches: [
-    { name: 'master', channel: 'latest' },
-  ],
```

Comment:
I assume we can delete this because the base .releaserc.js file takes precedence?